dkan.profile

```diff
@@ -501,7 +502,9 @@ edit-menu-description
     ))
     ->condition('eid', '5')
     ->execute();
-  drupal_flush_all_caches();
+}
+
+function dkan_post_install() {
```

Comment:
1. Drupal flush caches is not needed. 
2. Given we've been doing things in the last step like clear cache or enable aggregation, etc. I think it's worth no name this operation properly instead of doing that in another install operation. In other words: name last install step explicitly to avoid things like adding multiple clear cache at different operations and forget them later.